### PR TITLE
Page 0 host management features

### PR DESCRIPTION
**Pull request by retroinspect**
- Retrieving zones, hosts, and groups from cloud orchestrator
- Create Host view & Remaining of List Runtime view (creation / deletion of hosts)
- Long polling is TODO after MVP

### CODE REVIEW

File: web/page0/src/app/runtime-card/runtime-card.component.ts [hunk 1]
```diff
@@ -11,20 +15,61 @@ import { RuntimeService } from '../runtime.service';
...
+      .deleteHost(host.url)
+      .pipe(takeUntil(this.ngUnsubscribe))
+      .subscribe({
+        next: () => {
+          this.snackBar.dismiss();
+        },
+        error: (error) => {
+          this.snackBar.open(
+            `Failed to delete host ${host.url} (error: ${error.message})`
+          );
```
Comment (k311093) [inline web/page0/src/app/runtime-card/runtime-card.component.ts:71]: How about adding dismissal for snackbar?

https://material.angular.io/components/snack-bar/overview#dismissal

File: web/page0/src/app/runtime.service.ts [hunk 2]
```diff
@@ -188,28 +194,79 @@ export class RuntimeService {
...
+  }
+
+  private getHosts(
+    runtimeUrl: string,
+    zone: string,
+    runtimeAlias: string
+  ): Observable<Host[]> {
+    return this.apiService.listHosts(runtimeUrl, zone).pipe(
+      mergeMap(({ items: hosts }) => {
+        return forkJoin(
```
Comment (k311093) [inline web/page0/src/app/runtime.service.ts:210]: The statement `() => return forkJoin(....)` can be replaced with `() => forkJoin(....)`

**Review by k311093 — CHANGES_REQUESTED** [review]

File: web/page0/src/app/runtime-card/runtime-card.component.ts (hunk 1 above)
Comment (retroinspect) [reply web/page0/src/app/runtime-card/runtime-card.component.ts]: Done

File: web/page0/src/app/runtime.service.ts (hunk 2 above)
Comment (retroinspect) [reply web/page0/src/app/runtime.service.ts:211]: Parentheses are left to add console.log in handy.
Will apply on lint options all at once if available.

Comment (k311093) [reply web/page0/src/app/runtime.service.ts:211]: Acknowledged. Thank you.

**Review by k311093 — APPROVED** [review]

**Pull request merged**